### PR TITLE
Adding Student loans and Account typings

### PR DESCRIPTION
**Pull request by ossiggy**
- Adds new student loans type along with updates from [this PR](https://github.com/MethodFi/method-python/pull/18)
- Removes deprecated Connections

### CODE REVIEW

File: method/resources/Account.py [hunk 1]
```diff
@@ -34,35 +75,295 @@
...
+	expected_payoff_date: Optional[str]
+	delinquent_status: Optional[str]
+	delinquent_amount: Optional[int]
+	delinquent_period: Optional[int]
+	delinquent_action: Optional[str]
+	delinquent_start_date: Optional[str]
+	delinquent_major_start_date: Optional[str]
+	delinquent_status_updated_at: Optional[str]
+	delinquent_history: Optional[List[DelinquencyHistoryItem]]
+	delinquent_action: Optional[List[TrendedDataItem]]
```
Comment (stephenluc) [inline method/resources/Account.py:230]: nit: fix indentations

File: method/resources/Account.py [hunk 2]
```diff
@@ -34,35 +75,295 @@
...
+
+class AccountLiabilityPersonalLoan(AccountLiabilityLoan):
+  expected_payoff_date: Optional[str]
+  available_credit: Optional[int]
+  principal_balance: Optional[int]
+  year_to_date_interest_paid: Optional[int]
+
+
+class AccountLiabilityCreditBuilder(AccountLiabilityLoan):
+    pass
```
Comment (stephenluc) [inline method/resources/Account.py:270]: nit: indentation

File: method/resources/Account.py [hunk 3]
```diff
@@ -34,35 +75,295 @@
...
+
+
+class AccountLiabilityCollection(AccountLiabilityLoan):
+   pass
```
Comment (stephenluc) [inline method/resources/Account.py:274]: nit: indentation

**Review by stephenluc — CHANGES_REQUESTED** [review]
Looks like indentations are inconsistent throughout the files, can we fix those up. Additionally there looks like there's a merge conflict can you resolve that too

**Review by jmdelcarmen — DISMISSED** [review]

File: method/resources/Transaction.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    billing_amount: str
+    billing_currency: str
+    status: str
+    created_at: str
+    updated_at: str
+
+
+class TransactionResource(Resource):
+    def __init__(self, config: Configuration):
+        super(TransactionResource, self).__init__(config.add_path('resource'))
```
Comment (mustafalii) [inline method/resources/Transaction.py:24]: is this correct? 
`config.add_path('resource')`

Comment (jmdelcarmen) [reply method/resources/Transaction.py:24]: @ossiggy – should be config.add_path('transactions')

**Review by jmdelcarmen — APPROVED** [review]

**Review by stephenluc — APPROVED** [review]

**Pull request merged**